bing_rewards/__init__.py

```diff
@@ -310,20 +310,27 @@ def search(count, words_gen: Generator, agent, args, config):
         # Get a random query from set of words
         query = next(words_gen)
 
+        # Determine the key combination to use based on the --bing flag
+        if args.bing:
+            # Ctrl + E to open address bar wit the default search engine
+            key_combo = (Key.ctrl, "e")
+        else:
+            # Alt + D focuses address bar without using search engine
+            key_combo = (Key.alt, "d")
+
         # If the --bing flag is set, type the query to the address bar directly
         if args.bing:
             search_url = query
         else:
             # Concatenate url with correct url escape characters
             search_url = (config.get("search-url") or URL) + quote_plus(query)
 
-        # Use pynput to trigger keyboard events and type search querys
+        # Use pynput to trigger keyboard events and type search queries
         if not args.dryrun:
-            # Alt + D to focus the address bar in most browsers
-            key_controller.press(Key.alt)
```

Comment:
Just learned there is a `key_controller.pressed()` context manager from pynput, this might be a good place to use that.
https://pynput.readthedocs.io/en/latest/keyboard.html#controlling-the-keyboard